.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,9 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```

Comment:
Is this added intentionally? It could come in useful for folks.

Reply:
Yes! Precisely why I added it. :)